core/src/repair/cluster_slot_state_verifier.rs

```diff
@@ -853,7 +853,7 @@ pub(crate) fn check_slot_agrees_with_cluster(
     purge_repair_slot_counter: &mut PurgeRepairSlotCounter,
     slot_state_update: SlotStateUpdate,
 ) {
-    info!(
+    error!(
```

Comment:
hmm why make this error? This gets printed for every slot, i think it would be better if we only error on `SlotStateUpdate::Duplicate`

Reply:
lmao yup, all leftover from debugging, all reverted